charts/forms-flow-data-analysis/templates/deployment.yaml

```diff
@@ -21,6 +21,20 @@ spec:
         app: "{{ .Chart.Name }}"
     spec:
       containers:
+      - name: filebeat-data-analysis
```

Comment:
@amal-k-soman-aot @sinto-aot we need to make sure filebeat sidecar is optional

Reply:
sure